### ISSUE TITLE
What DL framework can we use in the testing phase?

### ISSUE THREAD

**Issue by lime-j** [amumu1843/Robust_Raw_Object_Detection#1]
Hello! Thank you for your interesting work! I have noticed that during the testing phase, we should submit our code to you, according to the instruction, "Our testing environment is a single 32G-memory GPU, with python 3.7.10 and CUDA 10.2. Please make sure the submitted codes are compatible with this environment". I wonder what DL framework can we use for the contest, PyTorch? TensorFlow? Mindspore?.

**Comment by amumu1843** [comment]
Please choose your framework from PyTorch, TensorFlow or MindSpore. All three frameworks are acceptable. I will update such information in the readme file. Thanks for your reminding.

**Comment by amumu1843** [comment]
@lime-j README is updated, and I also add some more information there. Please check it again.

**Comment by lime-j** [comment]
Thanks for your clarification :)

**Comment by lime-j** [comment]
One more question, will the leaderboard on the EvalAI platform be opened before the deadline?

**Comment by amumu1843** [comment]
@lime-j As far as I know, status of leader board cannot be changed once the challenge is launched. Alternatively, I will take a screenshot of the leader board on Aug. 18th and Aug. 25th (this and next Friday), and post them in this repository. Notice that the final ranking is **ONLY** based on the pure blind testing. Performance on validation set will **NOT** contribute to the final score.

**Comment by lime-j** [comment]
Thanks for your clarification, again :)
